lib/common.js

```diff
@@ -68,18 +69,9 @@ exports.hasProperties = function(obj) {
 
 exports.readJSON = function(file) {
   return asp(fs.readFile)(file)
-  .then(function(pjson) {
-    pjson = pjson.toString();
-    // remove any byte order mark
-    if (pjson.startsWith('\uFEFF')) // TODO: https://www.npmjs.com/package/strip-bom
-      pjson = pjson.substr(1);
-    try {
-      return JSON.parse(pjson);
-    }
-    catch(e) {
-      throw 'Error parsing package.json file ' + file;
```

Comment:
Yeah, I've added https://github.com/jspm/jspm-cli/pull/572/files#diff-1ffb5a9090758229a335b02a477425c7R78 just after you posted comment.